Package/PackageActions/List.cs

```diff
@@ -197,7 +197,7 @@ private void PrintReadable(List<PackageDef> packages, HashSet<PackageDef> instal
                 var installedPackage = installed.FirstOrDefault(p => p.Name == plugin.Name);
                 var latestPackage = packages.Where(p => p.Name == plugin.Name).OrderByDescending(p => p.Version).FirstOrDefault();
 
-                var installedString = installedPackage == null ? "" : " - installed";
+                var installedString = installedPackage == null ? "" : installedPackage.IsValid() ? " - installed" : " - needs reinstall";
                 
                 if (installedPackage != null && installedPackage.IsSystemWide())
                     installedString += " system-wide";
```

Comment:
I think this looks pretty bad:

```
PathWave License Manager         - 7.4.1+efeba870                                   - needs reinstall system-wide
```

Could we make it something like
```
PathWave License Manager         - 7.4.1+efeba870                                   - system-wide - needs reinstall
```

I'm not sure how I feel about the wording of "needs reinstall". I would prefer something like "not validated". What do you think?